chia/wallet/puzzles/nft_ownership_layer.clvm

```diff
@@ -67,25 +76,23 @@
               )
             )
             (if (= (f (f conditions)) CREATE_PUZZLE_ANNOUNCEMENT)
-                (assert (all
-                          (not (= (substr (f (r (f conditions))) 0 8) ANNOUNCEMENT_PREFIX))
+                (assert (not (and
                           (= 40 (strlen (f (r (f conditions)))))
-                        )
+                          (= (substr (f (r (f conditions))) 0 8) ANNOUNCEMENT_PREFIX)  ; lazy eval
```

Comment:
Sorry this is actually incorrect, this loop is assuming `(f (r` already (which I will fix) but the second argument to a condition may not be 8 bytes so `substr` will fail on small announcements.

Reply:
Er, again, I'm incorrect, I don't need to check `(f (r` because we already know this is a `CREATE_PUZZLE_ANNOUNCEMENT` which we can assume to have an arg

Reply:
oh yes, my b